lib/charms/opensearch/v0/helper_conf_setter.py

```diff
@@ -2,6 +2,7 @@
 # See LICENSE file for licensing details.
 
 """Utilities for editing yaml config files at any depth level and maintaining comments."""
+import logging
```

Comment:
Yes, so the reason I'm using `rueaml.yaml` is because of the fact it allows keeping and writing comments on a yaml doc, whereas PyYaml loses comments on load. OpenSearch has many yaml files with comments serving as docs, so i wanted to keep those comments

Reply:
Good idea for the generic config setter lib!